index.html

```diff
@@ -12,12 +12,13 @@
 			<label>Language Translator</label>
 			<textarea id="userinput" name="User Input" cols="20" rows="10"></textarea>
 			<select>
-				<option value="Select Language">Select Language</option>
-				<option value="Greek">Greek</option>
-				<option value="Japanese">Japanese</option>
-				<option value="Latin">Latin</option>
+				<div>
+					<option selected="selected" value="1">Greek</option>
+					<option value="2">Japanese</option>
+					<option value="3">Latin</option>
+				</div>
 			</select>
-			<button id="translate" type="button" onclick="translate()">Translate</button>
+			<button id="translate" type="button" onclick="getTextArea()">Translate</button>
```

Comment:
Okay, just to clarify, it just shows the <button> twice?

Sent from my iPhone

> On Apr 26, 2016, at 10:40 PM, Jessica Wynn <notifications@github.com> wrote:
> 
> In index.html:
> 
> >  			</select>
> > -			<button id="translate" type="button" onclick="translate()">Translate</button>
> > +			<button id="translate" type="button" onclick="getTextArea()">Translate</button>
> There are not 2 buttons. The red shows I where removed something to that button and the green shows where I added something to that button. I just renamed the function.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>